src/main/java/org/isf/accounting/service/AccountingIoOperations.java

```diff
@@ -329,24 +342,55 @@ public int compare(Bill o1, Bill o2) {
 	 * @param patient
 	 * @return
 	 * @throws OHServiceException
+	 * @deprecated use {@link #getPaymentsForPatient(GregorianCalendar, GregorianCalendar, Patient)}
 	 */
+	@Deprecated
 	public ArrayList<BillPayments> getPayments(GregorianCalendar dateFrom, GregorianCalendar dateTo, Patient patient)
 			throws OHServiceException {
+		return getPaymentsForPatient(dateFrom, dateTo, patient);
+	}
+
+	/**
+	 * Retrieves all billPayments for a given patient in the period dateFrom -> dateTo
+	 * @param dateFrom
+	 * @param dateTo
+	 * @param patient
+	 * @return
+	 * @throws OHServiceException
+	 */
+	public ArrayList<BillPayments> getPaymentsForPatient(GregorianCalendar dateFrom, GregorianCalendar dateTo, Patient patient)
```

Comment:
Nice the idea to add @deprecate, for this method I would suggest `getPaymentsBetweenDatesWherePatient`